### PR TITLE
✨style: improve visibility of invalid port number warning message

### PR DESCRIPTION
**Pull request by noejunior792**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default {
       } else if (this.portStatus.includes("closed")) {
         return { color: "red" };
       } else {
-        return { color: "yellow" };
+        return { color: "black" };
```
Comment (euotiniel) [inline src/App.vue:22]: Black? Do you can to send a picture?

**Review by euotiniel — CHANGES_REQUESTED** [review]
Submit a screenshot with this change, please.

File: src/App.vue (hunk 1 above)
Comment (noejunior792) [reply src/App.vue:22]: Captura de ecrã 2024-08-06 074322
Well, I'm not a good designer, but I understand how it works and for me a good design has to be functional, that's why I chose the color black.

Comment (euotiniel) [reply src/App.vue:22]: @noejunior792 Try to experiment with different shades of yellows.

Comment (noejunior792) [reply src/App.vue:22]: Captura de ecrã 2024-08-08 163017
Like this?

Comment (euotiniel) [reply src/App.vue:22]: What did you think?

Comment (noejunior792) [reply src/App.vue:22]: Better than the previous one, can I make a pull request with the change?

Comment (euotiniel) [reply src/App.vue:22]: Undoubtedly.

**Pull request closed without merging**